### PR TITLE
Teeth 89

### PR DESCRIPTION
**Pull request by kuroneko1011**

### CODE REVIEW

File: src/containers/ClinicDetailContainer/ClinicDetailContainer.container.jsx [hunk 1]
```diff
@@ -41,11 +43,16 @@ const ClinicDetailContainer = ({ clinicId }) => {
     getClinic();
     getClinicFeedback();
   }, []);
+
+  const onClick = (serviceId) =>
+    history.push(RoutePath.CLINIC_PAGE + "/" + clinicId + "/" + serviceId);
```
Comment (ZERO-94) [inline src/containers/ClinicDetailContainer/ClinicDetailContainer.container.jsx:48]: please use url param and generate path of react router dom

File: src/containers/ServiceDetailContainer/ServiceDetail.container.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  const [service, setService] = useState({});
+
+  const getService = async () => {
+    try {
+      const { data } = await getServiceById(serviceId);
+      setService(data);
+    } catch (e) {
+      notification["error"]({
+        message: `Something went wrong! Try again latter!`,
+        description: `There is problem while filter, try again later`,
```
Comment (ZERO-94) [inline src/containers/ServiceDetailContainer/ServiceDetail.container.jsx:19]: filter ? or fetch data ?

File: src/routers/Path.js [hunk 3]
```diff
@@ -6,6 +6,8 @@ const RoutePath = {
   INTERNAL_ERROR_PAGE: "/internal-error",
   LOGIN_PAGE: "/login",
   FEEDBACK: "/feedbacks",
+  SERVICE_ENDPOINT: "/services",
+  SERVICE: "/clinic/:clinicId/:serviceId",
```
Comment (ZERO-94) [inline src/routers/Path.js:10]: excuse me wtf is this var ? where is my PAGE word

File: src/services/teeth-apis/ServiceController.js [hunk 4]
```diff
@@ -1,3 +1,7 @@
+import RoutePath from "../../routers/Path";
 import { CallAPI } from "./axiosBase";
 
 export const getAllServices = () => CallAPI("/services", "GET");
+
+export const getServiceById = (serviceId) =>
+  CallAPI(RoutePath.SERVICE_ENDPOINT + "/" + serviceId, "GET");
```
Comment (ZERO-94) [inline src/services/teeth-apis/ServiceController.js:7]: this endpoint will be in other folder so please check it

File: src/components/customized-components/ClinicDetailComponent/ClinicDetail.component.jsx [hunk 5]
```diff
@@ -63,6 +62,7 @@ const ClinicDetailComponent = ({ currentClinic, feedback }) => {
                   src="http://australiandentalclinic.vn/wp-content/uploads/2017/02/teeth-whitening-sample1.jpg"
                 />
               }
+              onClick={() => onClick(service.id)}
```
[the remark this answers is not in the document]
Comment (kuroneko1011) [reply src/components/customized-components/ClinicDetailComponent/ClinicDetail.component.jsx:65]: becuz id only exists on service, and service depend on the currentClinic we pass to component

File: src/routers/Path.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (kuroneko1011) [reply src/routers/Path.js:10]: to levarage useParams

**Pull request merged**